### DISCUSSION TITLE
Unshort not working with url like this anymore
https://bbc.in/47vusYL. It's only for me?

### DISCUSSION THREAD

**Discussion by mrtaxi** [TrianguloY/URLCheck#307] [General]
Unshort not working with url like this anymore
https://bbc.in/47vusYL. It's only for me?

**Comment by PabloOQ** [comment]
Sadly that has been the case for a while, the unshorten.me service is currently broken, it usually doesn't work. It's a real shame because before it broke it worked really well.

Related #298
I thought we had an issue about unshorten.me not working but it seems I was wrong... Well I guess linking this to that issue works too.

**Reply by mrtaxi** [reply]
Thank you thanks for the detailed explanation!
